### DISCUSSION TITLE
专业版邮箱收不到验证码

### DISCUSSION THREAD

**Discussion by chiwenda** [getlantern/lantern-forum-cn#1075] [General]
购买了一年专业版，登录网易邮箱，收不到邮件验证码！！！

**Comment by MaxMKMax** [comment]
@chiwenda 您好. 请您提供我账号邮箱和支付单据. 谢谢您支持蓝灯.

**Comment by chiwenda** [comment]
***@***.***








---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月10日 01:48 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 专业版邮箱收不到验证码 (Discussion #1075) |

@chiwenda 您好. 请您提供我账号邮箱和支付单据. 谢谢您支持蓝灯.

**Comment by chiwenda** [comment]
什么情况，



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月10日 01:48 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 专业版邮箱收不到验证码 (Discussion #1075) |

@chiwenda 您好. 请您提供我账号邮箱和支付单据. 谢谢您支持蓝灯.

**Comment by yjyyzm** [comment]
我买了2年，也是收不到验证码，被骗了
